contrib/yolov5/REAMDE.md

```diff
@@ -0,0 +1,49 @@
+# Yolov5 support for Rikai
+## Usage
```

Comment:
Could you have a follow up ticket to mention:

1. this is pytorch yolov5
2. and which implementation of yolov5